src/components/specs/LaunchCard.spec.js

```diff
@@ -0,0 +1,21 @@
+import { create } from 'react-test-renderer';
+import LaunchCard from '../LaunchCard';
+
+jest.mock('react-router-dom', () => ({ Link: 'Link' }));
+jest.mock('../styledComponents', () => ({
+  Card: 'Card',
+  Content: 'Content',
+  FailedStatus: 'FailedStatus',
+  LaunchDate: 'LaunchDate',
+  MissionDetails: 'MissionDetails',
+  MissionName: 'MissionName',
+  SuccessStatus: 'SuccessStatus',
+  UpcomingStatus: 'UpcomingStatus',
+}));
+
+describe('LaunchCard', () => {
+  it ('renders', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
  it ("renders", () => {
```